sufficient_statistics.cc

```diff
@@ -72,6 +78,8 @@ double SufficientStatistics::GetHomozygousStatistic(const TrioModel &params) {
  */
 double SufficientStatistics::GetMismatchStatistic(const TrioModel &params) {
   return GetSequencingErrorStatistic(params, 2);
+    //Note: Circular dependence: SetReadDependentData(data_vec) in Update().  read_dependent_data() here
+    //Note: GetMismatches(heter/homo) can factor out, it's constant for given dataset
```

Comment:
I don't understand what you mean. I added the matches as part of the ReadDependentData, so all we pass in now is the params.